gradle.properties

```diff
@@ -9,7 +9,7 @@ releasePluginVersion=2.6.0
 puppycrawlCheckstyleVersion=8.18
 ballerinaGradlePluginVersion=0.14.1
 
-ballerinaLangVersion=2201.2.0
+ballerinaLangVersion=2201.3.0-20221009-071700-3fa663b4
```

Comment:
Distribution version in toml is not changed

Reply:
Updated in https://github.com/ballerina-platform/module-ballerina-log/pull/674/commits/7a39a4e4044f70833387526f29a2cc3b780a6669